### DISCUSSION TITLE
Linux kernel for macbooks with t2 chip (t2linux project)

### DISCUSSION THREAD

**Discussion by privalov-connect** [elementary/os#685] [Q&A]
I use elementary as second system on macbook 16 2019 intel. Need an external keyboard to install. Is it possible to build an image by throwing a kernel from the t2linux project? I tried to create packages.chroot with deb package t2 kernel, but the assembly fails with errors.

**Comment by ryonakano** [comment]
Sorry, we have no plan to add support for a kernel image with a specific hardware at the moment.
